fhservices.cfg.j2

```diff
@@ -57,26 +57,26 @@ define host {
        host_name {{ rhmap_router_dns }}
 }
 
-{% for fhservice in fh_services_ping %}
+{% for fhservice in fh_services if 'ping' in fhservice.get("checks") %}
```

Comment:
Alright, so the `/start` script should fail and the container keep restarting.
For the ones that have no default value, can we use the direct key access as we had before? That would help us detect eventual problems earlier.